charts/rstudio/templates/NOTES.txt

```diff
@@ -15,6 +15,11 @@ You can retrieve Rstudio URL via this command:
   export RSTUDIO=$(oc get route --namespace {{ .Release.Namespace }} -o yaml | yq '.items[0].spec.host')
```

Comment:
missing route name

Reply:
Yes. It assumes you don't have other routes already deployed. With your namespace, it'll return other values.
Need to find a better way for this one. With annotations maybe 🤔

Reply:
You know the name of the route before hand, isn't it? Why not use that?

Reply:
```sh
$ oc get route rstudio -n test-helm-charts -o yaml | yq .spec.host
"pedropedro.2.rahtiapp.fi.2.rahtiapp.fi"
```

Reply:
With this update I know. It wasn't the case before. I'll do the change

Reply:
Pushed new commit